sanskrit_parser/lexical_analyzer/sandhi_rules/ach_sandhi.txt

```diff
@@ -34,10 +34,10 @@
 
 # यण्-सन्धिः
 # इको यणचि (६.१.७७)
-[*हल्][इ, ई] + [*अच् - इई][*हल्+ं] = {0}य्{2}{3}
-[*हल्][उ, ऊ] + [*अच् - उऊ][*हल्+ं] = {0}व्{2}{3}
-[*हल्][ऋ, ॠ] + [*अच् -ऋॠऌ][*हल्+ं] = {0}र्{2}{3}
-[*हल्][ऌ]+ [*अच् -ऋॠऌ][*हल्+ं] = {0}ल्{2}{3}
```

Comment:
I thought these changes were for handling the case of "इत्य् अपि". No harm adding them I suppose.

Reply:
These particular ones were for ity api - I neglected to remove them. 
It's lines 46-51 that are for this one. 
```# Need to handle जश्त्वसन्धिः
-त् + [ट्, ठ्, ड्, ढ्, ण्] = ट्{1} / ड्{1}
-थ् + [ट्, ठ्, ड्, ढ्, ण्] = ठ्{1} / ड्{1}
-द् + [ट्, ठ्, ड्, ढ्, ण्] = ड्{1} 
-ध् + [ट्, ठ्, ड्, ढ्, ण्] = ढ्{1} / ड्{1}
-न् + [ ट्, ठ्, ड्, ढ्, ण्] = ण्{1} / ड्{1}
```